src/test/java/com/es/phoneshop/model/product/CartItemTest.java

```diff
@@ -0,0 +1,71 @@
+package com.es.phoneshop.model.product;
+
+import com.es.phoneshop.model.product.cart.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.util.Currency;
+
+import static org.junit.Assert.*;
+
+public class CartItemTest {
+    private ProductDao productDao;
+    private CartService cartService;
+    Currency usd = Currency.getInstance("USD");
+    int quantity;
+
+    @Before
+    public void setup() {
+        productDao = ArrayListProductDao.getInstance();
+        cartService = DefaultCartService.getInstance();
+    }
+
+    @Test
+    public void testAddProductToCart() throws OutOfStockException {
+        Product product = new Product("test-product", "Apple iPhone", new BigDecimal(200), usd, 10, "https://raw.githubusercontent.com/andrewosipenko/phoneshop-ext-images/master/manufacturer/Apple/Apple%20iPhone.jpg");
+        productDao.save(product);
+        assertNotNull(product.getId());
```

Comment:
you can remove this assertion, because you already have test for DAO, where it's checked